test_frontend_simulation.py

```
@@ -0,0 +1,157 @@
+                        data = await response.json()
+                        print(f"✅ Respuesta exitosa en {response_time:.2f}ms:")
+                        print(f"   Agente usado: {data.get('agent_used', 'N/A')}")
+                        print(f"   Respuesta: {data.get('response', '')[:150]}...")
+                        
+                        # Verificar si hay FAQs encontradas
+                        metadata = data.get('metadata', {})
+                        faq_results = metadata.get('faq_results', [])
+                        if faq_results:
+                            print(f"   FAQs encontradas: {len(faq_results)}")
+                        else:
+                            print(f"   FAQs encontradas: 0 (usando respuesta mock)")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)